PR Check Unsuccessful

I pulled a request [#307](https://github.com/scikit-hep/boost-histogram/pull/307) to fix some bugs in the example codes and document. But git shows that the check is unsuccessful, plz help me with this problem! 

```
Run actions/setup-python@v11s
3.8.1 (x64)
Run actions/setup-python@v1
##[error]Version 3.8.0 with arch x64 not found
Available versions:

2.7.17 (x64)
3.5.9 (x64)
3.6.10 (x64)
3.7.6 (x64)
3.8.1 (x64)
```

--- a/.github/workflows/examples.yml
+++ b/.github/workflows/examples.yml
@@ -18,7 +18,7 @@ jobs:
         submodules: true
     - uses: actions/setup-python@v1
       with:
-        python-version: '3.8.0'
+        python-version: '3.8'
     - name: Pre-install Numpy
       run: python -m pip install numpy nbconvert ipykernel
     - name: Install kernel

--- a/setup.py
+++ b/setup.py
@@ -130,7 +130,7 @@ class BuildExt(build_ext):
 extras = {
     "test": ["pytest", "pytest-benchmark", "numpy", 'futures; python_version < "3"'],
     "docs": ["Sphinx>=2.0.0", "recommonmark>=0.5.0", "sphinx_rtd_theme"],
-    "examples": ["numpy", "ipykernel", "matplotlib"],
+    "examples": ["numpy", "ipykernel", "matplotlib", "xarray"],
 }
 
 setup(

